jsk_fetch_robot/jsk_fetch_startup/scripts/safe_tilt_head.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2017 JSK Robotics Lab.
+# Copyright (c) 2015 Fetch Robotics Inc.
```

Comment:
please add 
```
this code is originally copied from https://github.com/fetchrobotics/fetch_ros/blob/0.7.12/fetch_navigation/scripts/tilt_head.py,
We need to re-write this code because
- ...
- ...
```

Reply:
@k-okada OK. Added: https://github.com/jsk-ros-pkg/jsk_robot/pull/834/files#diff-ca861c075b7503345f0e3399b5cab010R34